embassy/src/time/duration.rs

```diff
@@ -5,6 +5,7 @@ use super::TICKS_PER_SECOND;
 
 #[derive(Debug, Default, Copy, Clone, PartialEq, Eq, PartialOrd, Ord)]
 #[cfg_attr(feature = "defmt", derive(defmt::Format))]
+/// Represents the difference between [Instant::now()](struct.Instant.html#method.now) and some other Instant
```

Comment:
This doesn't necessarily represent the difference between now and another instant, it can be between any two instants.

Reply:
Ah. I made an assumption given Duration's usage in the delay abstractions. will correct.